ios/app/brave_core_main.h

```diff
@@ -44,7 +44,7 @@ OBJC_EXPORT
 
 - (void)scheduleLowPriorityStartupTasks;
 
-@property(class, readonly) id<BraveWalletERCTokenRegistry> ercTokenRegistry;
+@property(class, readonly) id<BraveWalletBlockchainRegistry> blockChainRegistry;
```

Comment:
tiny nit: If the type doesn't camel-case "block" and "chain", the property name probably shouldn't either

```suggestion
@property(class, readonly) id<BraveWalletBlockchainRegistry> blockchainRegistry;
```

Reply:
oops it should be blockchain, I'll rebase and fix.

Reply:
updated